pages/covid-19/guidelines.vue

```diff
@@ -1,11 +1,94 @@
 <template>
-    <p>COVID 19 Guidelines</p>
+    <i18n-page-wrapper class="covid19 px-10 md:px-20 lg:px-36 xl:px-60">
+        <core-h1 :title="$t('pageTitle')" class="covid19__pageTitle"></core-h1>
+        <div class="covid19__summary">
+            <p
+                v-for="(item, index) in $t('latestEditedTime')"
+                :key="`latestEditedTime_${index}`"
+            >
+                {{ item }}
+            </p>
```

Comment:
- Replace `flex justify-center` with [`text-right`](https://tailwindcss.com/docs/text-align) for this section.
- Translate only for those words that needed.
```suggestion
            <p>
                (CC BY-SA 3.0 TW)
                {{ $t('XXXX') }}: 2021-06-15
            </p>
```